### PR TITLE
create a user profile 

### PR DESCRIPTION
**Pull request by lys1k**

### CODE REVIEW

**Comment by Deutscher862** [comment]
There are some conflicts you need to resolve first

File: backend/src/main/java/application/model/user/User.java [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
+    @NotBlank
+    @Column(name= "email")
+    @Size(min = 6, max = 40)
+    @Email
+    private  String email;
+
+    @NotBlank
+    @Column(name = "password")
+    @Size(min = 8, max = 40)
+    private String password;
```
Comment (Deutscher862) [inline backend/src/main/java/application/model/user/User.java:41]: I assume we won't keep unencrypted password in the database. Or is it only temporary?

File: backend/src/main/java/application/payload/response/JwtResponse.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class JwtResponse {
+    private String token;
+    private final String type = "Bearer";
+    private Long id;
+    private String username;
+}
```
Comment (Deutscher862) [inline backend/src/main/java/application/payload/response/JwtResponse.java:13]: Git requires an empty line at the end of each file:
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

**Review by Deutscher862 — CHANGES_REQUESTED** [review]

File: backend/src/main/java/application/controller/TestController.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/test")
+public class TestController {
+    @GetMapping("/auth")
+    public String testAuth() {
+        return "Success!";
+    }
+}
```
[the remark this answers is not in the document]
Comment (lys1k) [reply backend/src/main/java/application/controller/TestController.java:14]: True.

File: backend/src/main/java/application/model/user/User.java (hunk 1 above)
Comment (lys1k) [reply backend/src/main/java/application/model/user/User.java:41]: Yes, that's my mistake.

**Pull request merged**